apps/bare-expo/package.json

```diff
@@ -67,13 +67,13 @@
       },
       "android.emu.debug": {
         "binaryPath": "android/app/build/outputs/apk/debug/app-debug.apk",
-        "build": "cd android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ..",
+        "build": "cd android && ./gradlew :app:assembleDebug :app:assembleAndroidTest -DtestBuildType=debug && cd ..",
```

Comment:
Would like to explain this more:
`unimodules-test-core` is also subproject of bare-expo.
If we do not specify the target, this will also build `unimodules-test-core:assembleAndroidTest`.
Inside unimodules-test-core, there is a `api 'io.mockk:mockk:1.9.3'` depending on newer android sdk.
The formal solution is to use mockk-android artifact for android instrument test:
```
testImplementation "io.mockk:mockk:{version}"
androidTestImplementation "io.mockk:mockk-android:{version}"
```
See: https://github.com/mockk/mockk/issues/281

However, the dependencies are getting a little hard to be fixed like the formal way.
So the thing is to prevent `unimodules-test-core:assembleAndroidTest` as unimodules-test-core is specific for unit test.